docs/dev-notes/BACKGROUND-QUEUE.md

```diff
@@ -13,22 +13,20 @@ By running tasks at a later time, there are many great benefits.
 
 Instead of designing our SDK to be asynchronous like this:
 
-```swift 
+```swift
 CustomerIO.shared.identify("Dana") { result in 
-    switch result {
-        case .success: 
-            // yay! Profile identified! 
-        case .failure(let cioError): 
-            // Handle failure. Determine what failed and why. 
-            // Write custom code to maybe retry the request later. 
+    // handle the result. If success or error 
+    switch (result) {
+        ...
```

Comment:
any particular reason to remove the `.success & .failure` case from the sample? seemed like a nice starting point to be able to copy/paste and add logic

Reply:
The code is for example purposes, only. The SDK function `identify()` does *not* contain a callback function anymore so there would not be a point in copy/paste. I tried to make the code shorter because it's now for demonstration purposes.